README.md

```diff
@@ -4,7 +4,69 @@ An API for friendly boolean parsing and handling
 
 ## Usage
 
-TODO: Write code samples and usage instructions here
+```ruby
+require 'much-boolean'
+MuchBoolean::FALSE_VALUES # => [nil, 0, "0",
+                                false, "false", "False", "FALSE", "f", "F",
+                                "no", "No", "NO", "n", "N"]
```

Comment:
@kellyredding - Not sure if you want to put comments in front of these? I'm not sure what's the preferred way to do multi-line return values like this. Anyway, just noticed the syntax wasn't highlighting it as a comment.